django/utils/html.py

```diff
@@ -100,6 +102,13 @@ def format_html(format_string, *args, **kwargs):
     and call mark_safe() on the result. This function should be used instead
     of str.format or % interpolation to build up small HTML fragments.
     """
+    if not (args or kwargs):
+        # RemovedInDjango60Warning: when the deprecation ends, replace with:
+        # raise ValueError("args or kwargs must be provided.")
```

Comment:
For my own understanding. Why this approach over changing the function signature?

Reply:
How you would change it? (positional or keyword arguments must be provided besides `format_string`) :thinking: